ethereum/WebaverseERC1155.sol

```diff
@@ -91,16 +91,16 @@ contract WebaverseERC1155 is
 
     /**
      * @return Returns the token URI against a particular token id.
-     * e.g. [baseURI]/[_id]
+     * e.g. It's just the token's content URL, which will most likely be a metaversefile directory. But it could also be a GH pages or similar.
```

Comment:
`contenturl` and `uri()` seems different.

Reply:
From avaer:
> there is the URL we set during mint which in our case will probably point to web3.storage
> that is not the same as the URL the contract returns for the API
> our API looks at the contract's url to download the metaversefile and serve the details
> but there are two entirely different URLS

Reply:
So should uri() return URL like "https://tokens.webaverse.com/1"?

Reply:
Yes